paper-badge.html

```diff
@@ -28,7 +34,7 @@
 
     <div>
       <paper-button id="btn">Status</paper-button>
-      <paper-badge for="btn" label="♥︎"></paper-badge>
+      <paper-badge icon-badge for="btn" label="favorite"></paper-badge>
```

Comment:
SGTM. That also means we can remove the `icon-badge` attribute as well. In terms of ARIA, one of the reasons I used `label` for the icon IDs originally was because their IDs are pretty semantic ("favorite", "home", etc.) so it would help for A11Y with icons. Based on the suggested changes, do you think icon badges should still have `aria-label`s, and if so what should they be set to? One idea I thought of was to chop off any custom iconset prefix and use the result e.g.

```html
<!-- aria-label here would be "business" -->
<paper-badge icon="communication:business" ...></paper-badge>
```